packages/core-cairo/src/build-generic.ts

```diff
@@ -2,12 +2,14 @@ import { ERC20Options, buildERC20 } from './erc20';
 import { ERC721Options, buildERC721 } from './erc721';
 import { ERC1155Options, buildERC1155 } from './erc1155';
 import { CustomOptions, buildCustom } from './custom';
+import { AccountOptions, buildAccount } from './account';
 
 export interface KindedOptions {
   ERC20:    { kind: 'ERC20' }    & ERC20Options;
   ERC721:   { kind: 'ERC721' }   & ERC721Options;
   ERC1155:  { kind: 'ERC1155' }  & ERC1155Options;
-  Custom:  { kind: 'Custom' }  & CustomOptions;
+  Custom:   { kind: 'Custom' }  & CustomOptions;
+  Account:  { kind: 'Account' }  & AccountOptions;
```

Comment:
Does this affect the order?
```suggestion
  Account:  { kind: 'Account' }  & AccountOptions;
  Custom:   { kind: 'Custom' }  & CustomOptions;
```